### ISSUE TITLE
Relative import vs absolute imports

### ISSUE THREAD

**Issue by uncojohnco** [alysivji/busy-beaver#65]
HIya,

Just wondering why `relative imports`<sup>[[1]](https://realpython.com/absolute-vs-relative-python-imports)</sup> are implemented rather than absolute? I've just had bad experiences in the past with relative imports in larger eco sytems :frowning:  
e.g. I wouldn't be surprised if eventually the `adapters` are split into their own repo while the `backend` and `models` are promoted as a "lower level library"?

[1] https://realpython.com/absolute-vs-relative-python-imports

**Comment by alysivji** [comment]
I use relative imports as this project grew organically from me playing around with the GitHub API. I've been able to change the directory structure fairly painlessly as I started working on additional features.

My workflow is fairly fluidly in terms of moving files and renaming variables as it makes sense. Tests make this easy to do. I think planning things out would increase structure and would reduce the need for relative imports.

I'm open to either style. What problems did you experience? Would love to learn more.
